Campus-iOS/RoomFinder/Views/RoomFinderDetailsBaseView.swift

```diff
@@ -27,7 +27,7 @@ struct RoomFinderDetailsBaseView: View {
                 Divider()
                 LectureDetailsBasicInfoRowView(
                     iconName: "building.columns",
-                    text: "\(room.campus), \(room.buildingNumber)"
+                    text: "\(String(describing: room.campus)), \(room.buildingNumber)"
```

Comment:
Since `room.campus` is an optional string, and in order to silence the warning:

<img width="757" alt="image" src="https://user-images.githubusercontent.com/59373377/236554296-921aad66-61fd-4c18-a172-8fec2bde1bb1.png">

It is now solved by providing a default value
<img width="601" alt="image" src="https://user-images.githubusercontent.com/59373377/236554747-3bd6f684-7d68-437e-88bd-231736acb0c2.png">